recipes/mpg123/all/conanfile.py

```diff
@@ -0,0 +1,189 @@
+from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class Mpg123Conan(ConanFile):
+    name = "mpg123"
+    description = "Fast console MPEG Audio Player and decoder library"
+    topics = ("conan", "mpg123", "mpeg", "audio", "player", "decoder")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://mpg123.org/"
+    license = "??"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "flexible_resampling": [True, False],
+        "network": [True, False],
+        "icy": [True, False],
+        "id3v2": [True, False],
+        "ieeefloat": [True, False],
+        "layer1": [True, False],
+        "layer2": [True, False],
+        "layer3": [True, False],
+        "moreinfo": [True, False],
+        "seektable": "ANY",
+        "module": ["dummy", "libalsa", "tinyalsa", "win32"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "flexible_resampling": True,
+        "network": True,
+        "icy": True,
+        "id3v2": True,
+        "ieeefloat": True,
+        "layer1": True,
+        "layer2": True,
+        "layer3": True,
+        "moreinfo": True,
+        "seektable": "1000",
+        "module": "dummy",
+    }
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake", "pkg_config", "cmake_find_package"
+
+    _autotools = None
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.options.shared:
+            del self.options.fPIC
+        try:
+            int(self.options.seektable)
+        except ValueError:
+            raise ConanInvalidConfiguration("seektable must be an integer")
+        if self.settings.os != "Windows":
+            if self.options == "win32":
+                raise ConanInvalidConfiguration("win32 is an invalid module for non-Windows os'es")
+
+    def requirements(self):
+        if self.options.module == "libalsa":
+            self.requires("libalsa/1.2.4")
+        if self.options.module == "tinyalsa":
+            self.requires("tinyalsa/1.1.1")
```

Comment:
https://sourceforge.net/p/mpg123/bugs/299/

Reply:
That's a me!